javascript/javascript2/week1/spiritAnimalName/spiritAnimalName.js

```diff
@@ -0,0 +1,69 @@
+const input = document.createElement('input');
+const btn = document.createElement('button');
+btn.textContent = 'click here';
+const body = document.querySelector('body');
+body.appendChild(input);
+body.appendChild(btn);
+
+// spirit animal list
+const spiritListAnimals = ['Mellow Mammoth', 'Numbat-Chihuahua', 'Focus Hawk', 'Guppy-Wapiti', 'Vampire Peacock', 'Hornbill-Duckbill', 'Platinum-Eyed Fossa', 'Timberwolf-Scaled Stoat', 'Glass Labradoodle', 'Ermine-Tarsiere'];
+const randomSpiritAnimalNames = spiritListAnimals[Math.floor(Math.random(spiritListAnimals) * 10)];
```

Comment:
In general you should not have constants that rely on arrays that can be modified
```suggestion
const randomSpiritAnimalNames = spiritListAnimals[Math.floor(Math.random(spiritListAnimals) * spiritListAnimals.size()];
```